EntAppProject/src/main/resources/templates/JBWelcomeScreen.html

```diff
@@ -2,35 +2,23 @@
 
 <html>
 

```

Comment:
I redid the HTML page restructuring from before but with the change of using the `page-title` class on span elements to specify that it should use the style included in `JobBookCSS.css`.